src/System.Private.CoreLib/shared/System/Environment.Win32.cs

```diff
@@ -29,7 +30,7 @@ private static string GetEnvironmentVariableFromRegistry(string variable, bool f
             }
         }
 
-        private static void SetEnvironmentVariableFromRegistry(string variable, string value, bool fromMachine)
+        private static void SetEnvironmentVariableFromRegistry(string variable, string? value, bool fromMachine)
```

Comment:
We haven't annotated `RegistryKey` but, `OpenEnvironmentKeyIfExists` can return null, if we want to avoid revisiting this file we should annotate the variables accordingly:

https://github.com/dotnet/coreclr/pull/23774/files#diff-5cde8619be34000aa795386e14743202R48

Reply:
Annotated.